modules/bene_media_file/src/Plugin/media/Source/File.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\bene_media_file\Plugin\media\Source;
+
+use Drupal\bene_media\FileInputExtensionMatchTrait;
+use Drupal\bene_media\InputMatchInterface;
+use Drupal\media\Plugin\media\Source\File as BaseFile;
+
+/**
+ * Input-matching version of the File media type.
+ */
+class File extends BaseFile implements InputMatchInterface {
+
```

Comment:
I'm still seeing a problem when I try to upload a file with a .doc extension. Now it says:

![screen shot 2018-02-26 at 9 52 40 am](https://user-images.githubusercontent.com/999525/36686405-2719c790-1adb-11e8-9bd6-35758c8b5ba8.png)

"6" is the file id I'm pretty sure. Creating this media at `/media/add/file` works fine.

Reply:
hm, I coulda sworn I had that fixed. I should be able to get this figured in the next week or so--just have to keep my head down on LVHN for the moment.